src/deploy/hosting/convertConfig.ts

```diff
@@ -136,8 +151,43 @@ export async function convertConfig(
       }
       const id = rewrite.function.functionId;
       const region = rewrite.function.region;
-      const endpoint = findEndpointForRewrite(deploy.config.site, haveBackend, id, region);
+      let endpoint: backend.Endpoint | undefined = undefined;
+      let functionIsInDeployingBackends = false;
+      for (const backend of wantBackends) {
+        const deployingEndpointSearch = findEndpointForRewrite(
+          deploy.config.site,
+          backend,
+          id,
+          region
+        );
+        functionIsInDeployingBackends =
+          functionIsInDeployingBackends || deployingEndpointSearch.endpointExistsSomewhere;
```

Comment:
I feel like "endpointExistsSomewhere" isn't super clear because the endpoint may not have existed somewhere (it could have been in the wrong region). I wonder if "foundMatchingId" or something similar might convey the technical meaning more crisply.

Reply:
Agreed. Renamed.